src/main/java/com/sfuentes/elections/Result.java

```diff
@@ -0,0 +1,52 @@
+package com.sfuentes.elections;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Result {
+
+  private final List<Vote> votes;
+
+  public Result(List<Vote> votes) {
+    this.votes = votes;
+  }
+
+  private Map<String, Integer> countVotes() {
+    int firstCandidateCount = 0;
+    int secondCandidateCount = 0;
+    int thirdCandidateCount = 0;
+    Map<String, Integer> candidates = new HashMap<>();
+    for (Vote vote : votes) {
+
+      switch (vote.getNameCandidate().toLowerCase()) {
+        case "mauricio" -> firstCandidateCount++;
+        case "joaquin" -> secondCandidateCount++;
+        case "rodrigo" -> thirdCandidateCount++;
+      }
+    }
+    candidates.put("Mauricio: ", firstCandidateCount);
+    candidates.put("Joaquin: ", secondCandidateCount);
+    candidates.put("Rodrigo: ", thirdCandidateCount);
+
+    return candidates;
+  }
+
+  public Map<String, Integer> getWinningCandidate() {
```

Comment:
This should be a class like ElectionResults.

Reply:
Fixed